php-frontend/src/main/java/org/sonar/php/parser/PHPLexicalGrammar.java

```diff
@@ -239,6 +239,7 @@
 
   SELF,
   PARENT,
+  STATIC,
```

Comment:
There is already a `STATIC` in `PHPKeyword.java`. I think we are fine when we use that one (just like was done for `ARRAY` and `CALLABLE` in the `TYPE_NAME()` rule.)